index.js

```diff
@@ -76,6 +76,8 @@ function filter (obj, { protoAction = 'error', constructorAction = 'error', safe
 
       if (constructorAction !== 'ignore' &&
           Object.prototype.hasOwnProperty.call(node, 'constructor') &&
+          node.constructor !== null &&
```

Comment:
can the same issue arise if __proto__ is null? i mean the code is basically the same just about 10 lines above

Reply:
I checked it. It does not throw any error, but the `__proto__` behaviour is different.